ha-singleton-service/README.md

```diff
@@ -3,44 +3,47 @@
 Author: Radoslav Husar  
 Level: Advanced  
 Technologies: MSC, Singleton Service, Clustering  
-Summary: The `ha-singleton-service` quickstart demonstrates deploying a cluster-wide singleton MSC service.
+Summary: The `ha-singleton-service` quickstart demonstrates how to deploy a cluster-wide singleton MSC service.  
```

Comment:
The extra spaces force a line feed. Without them, "Target Product" appears on the same line as "Sumary".